releasenotes/notes/sk-ignore-unsupported-ops-8d7d5f6fca255ffb.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade_transpiler:
+  - |
+    The :class:`.SolovayKitaev` transpiler pass no longer raises an exception on circuits
+    that contain single-qubit operations without a ``to_matrix`` method (such as measures,
+    barriers, control-flow operations) or parameterized single-qubit operations.
```

It might be good to state what happens with them 🙂 
```suggestion
    barriers, control-flow operations) or parameterized single-qubit operations,
    but will leave them unchanged.
```